Makefile

```diff
@@ -1,111 +1,115 @@
-# make    					build the entire site (all languages)
-# make en						build the English edition (replace 'en' with 'fr' for French)
+# make	 					build the entire site (all languages)
+# make init					just install NPM dependencies
+# make en					build just the English edition (replace 'en' with 'fr' for French, etc.)
 # make clean				destroy built files
-# make reset				destroy built files and build the entire site
-# make watch_css		watches for stylus (CSS) edits and compiles it
+# make reset				destroy built files and build all languages from scratch
+# make watch_css			watches for stylus (CSS) edits and compiles it
 
-default: clean_tmp mkdirs copy_assets build_all finalize
+#----------------------------------------------------------------------
+# CONFIGURATION
+#----------------------------------------------------------------------
 
-#---------------------------------------------------------------------- 
-# ALIASES 
-#---------------------------------------------------------------------- 
+# Collect list of languages based on available locale files
+LANGUAGES = $(shell find ./source/locales/ -name '*.json' -execdir basename -s .json {} \;)
 
-# Binaries
-BIN = ./node_modules/.bin/
-LIVESCRIPT_BIN = $(BIN)lsc
-LIVESCRIPT_PARAMS = -cob
-STYLUS_BIN = $(BIN)stylus
-STYLUS_PARAMS = -c -u ./node_modules/nib/
-STYLUS_WATCH_PARAMS = -c -w source/stylesheets/screen.styl -u nib -o public/assets/css/
+# Collect list of static assets that get copied over
+ASSETS = $(shell ls source/assets)
```

Comment:
To my knowledge only directories are expected there. This would adapt to new or renamed directories with no change to the `Makefile`, but if people start putting random stuff of other types it might need some adjustment. See 74eb936 for how I skinned that cat slightly differently before seeing your suggestion, but I think I prefer not even shelling out there and just using internal functions.